### PR TITLE
Develop jh

### PR DESCRIPTION
**Pull request by johannah-pik**
- Several improvements of CompScen report
- Include option to ship external sources for FE prices
- Include first version of ECEMF scenarios
- Delete old CompScen

### CODE REVIEW

File: inst/extdata/edget-mitigation.csv [hunk 1]
```diff
@@ -1,1601 +1,1985 @@
-SSP_scenario,tech_scenario,regioncat,level,subsector_L3,subsector_L2,FV_vehvar,FV_techvar,target,symmyr,speed
```
Comment (Loisel) [inline inst/extdata/edget-mitigation.csv:1]: Please do not replace commas by semicolons, diff becomes unreadable

File: inst/extdata/inconv_factor.csv [hunk 2]
```diff
@@ -1,513 +1,569 @@
-SSPscen,techscen,param,value
```
Comment (Loisel) [inline inst/extdata/inconv_factor.csv:1]: Same here. If you have problems with excel in that regard, please use R where you can control the format.

**Review by Loisel — CHANGES_REQUESTED** [review]

**Review by Loisel — APPROVED** [review]

File: R/reportEDGETransport2.R [hunk 3]
```diff
@@ -448,7 +448,7 @@ reportEDGETransport2 <- function(output_folder = ".",
     load_factor <- load_factor[, c("year", "region", "vehicle_type", "loadFactor", "technology")]
     demand_vkm <- merge(demand_km, load_factor, by = c("year", "region", "vehicle_type", "technology"))
     demand_vkm[, value := value / loadFactor] ## billion vkm
-    load_factor <- unique(loadFactor[year >= 1990, c("region", "year", "loadFactor", "vehicle_type")])
+    load_factor <- unique(load_factor[year >= 1990, c("region", "year", "loadFactor", "vehicle_type")])
```
Comment (Loisel) [inline R/reportEDGETransport2.R:451]: Does this mean the vintages reporting now works with both iterative and SA?

Comment (johannah-pik) [reply R/reportEDGETransport2.R:451]: Yes.. I implemented it already a few days ago but still received an error - turns out it was due to a typo in my bugfix :)

**Pull request merged**